### DISCUSSION TITLE
How to access the electronic gyroscope data of a smartwatch.

### DISCUSSION THREAD

**Discussion by vaas1993** [zepp-health/zeppos-forum#158] [Q&A]
I need to collect data from the electronic gyroscope to improve the functionality of the app.

**Comment by MyoungXUE** [comment]
Hi, the gyroscope API will be released in Zepp OS 3.0 and is not available yet.

https://docs-testing.zepp.com/docs/next/reference/device-app-api/newAPI/sensor/Gyroscope/
